20180125/MINUTES.md

```diff
@@ -0,0 +1,132 @@
+# Sardana Follow-up Meeting - 2018/01/25
+
+Held on 2018/01/25 at 14:00
+
+## Agenda
+
+1. Presentation
+  1.  Sardana objects references and the compatibility with Taurus4 (#668 - kick off)
+  2.  Centralized logging for Sardana with Elastic
+2. User Experience Report - Round Table
+3. Collaborative Developments - Review
+4. New Developments, Enhancements or Bug Fixes
+5. Pending PR distribution and Jan17 release
+6. Any Other Business
+
+## Minutes
+
+### 1. Presentation
+#### 1.1  Sardana objects references and the compatibility with Taurus4
+
+- PR: [#668](https://github.com/sardana-org/sardana/pull/668)
+- Because of the cache mechanism if you change the alias, you must make an init in the pool.
+- Keep reference able to distinguish devices and not relying on the tango (FQTN).
+- What if the tango host changes? Problem with memorized attributes. PR: [#667](https://github.com/sardana-org/sardana/issues/667)
+- Keep ID's and references internally
+- Internally, tango always uses FQTN. It would be good to keep the names unified.
```

Comment:
* Again FQDN.
* This is not always true e.g. `Device_4Impl.init_device` receives short name only e.g. without authority and withoud scheme (which in case of Tango is obvious).